srcjs/handlers/updateApp.js

```diff
@@ -5,6 +5,42 @@ $(function() {
 
   // Update app instance params (see updateF7App)
   Shiny.addCustomMessageHandler("update-app", function(message) {
+    // If color, dark or theme is changed, we need to update it
+    if (message.hasOwnProperty("color")) {
+      app.setColorTheme(message.color);
+    }
+
+    if (message.hasOwnProperty("dark")) {
+      if (message.dark) {
+        $("html").addClass("dark");
```

Comment:
Isn’t app.setDarkTheme working? Maybe not, I don’t remember… https://framework7.io/docs/app#method-app.setDarkMode(mode).

Reply:
@DivadNojnarg no, it doesn’t do anything. It should work, but if you try in console it changes some classes but it doesn’t change appearance. Might be a bug too?

Reply:
That being said, they use it in the Kitchensink example, but I noticed they call .$update() after that and I have no idea where that comes from 🤔

Reply:
I tried, it didn't work, but I just tried it again, and it does work 😬. So yeah, replaced with `setDarkMode`

Reply:
cache?

Reply:
No, I tested on a tab layout yesterday, and tested on a single layout today. With the tab layout, it does not work. So I think I'm going to change it back?